### DISCUSSION TITLE
I didn't follow the uninstall steps

### DISCUSSION THREAD

**Discussion by taso-kats** [Whisky-App/Whisky#885] [Q&A] [answered]
I didn't follow the uninstall steps, and it looks like I have 60plus GB somewhere hidden - searching for 'Bottles' doesn't show up. 
Any ideas to help recover the space so that I can do a clean install of Whisky?

**Comment by IsaacMarovitz** [comment, marked as answer]
Please see the instructions here: https://docs.getwhisky.app/paths.html#uninstalling-whisky
